doc/whats_new/v0.23.rst

```diff
@@ -341,7 +341,8 @@ Changelog
   pandas sparse DataFrame.
   :pr:`16021` by :user:`Rushabh Vasani <rushabh-v>`.
 
-- |Feature| :func:`utils.extmath.randomized_svd` now accepts 
+- |Feature| :func:`utils.extmath.randomized_svd` and 
```

Comment:
Tests are still failing. Also please add a test for `randomized_svd` on a LinearOperator.

Reply:
Fixed. The test added